content/en/tracing/setup_overview/compatibility_requirements/dotnet-framework.md

```diff
@@ -21,21 +21,27 @@ The .NET Tracer supports all .NET-based languages (for example, C#, F#, Visual B
-| 4.6.1                   | 04/26/2022            | [GA](#support-ga)                   | latest          |                     |
-| 4.6                     | 04/26/2022            | [Maintenance](#support-Maintenance) | < 2.0.0         | 04/26/2022          |
-| 4.5.2                   | 04/26/2022            | [Maintenance](#support-Maintenance) | < 2.0.0         | 04/26/2022          |
-| 4.5.1                   | 01/12/2016            | [Maintenance](#support-Maintenance) | < 2.0.0         | 04/26/2022          |
-| 4.5                     | 01/12/2016            | [Maintenance](#support-Maintenance) | < 2.0.0         | 04/26/2022          |
-
-Additional information can be found within [Microsoft's .NET Core Lifecycle Policy][3] and in [Runtime support policy for .NET Framework APM](#runtime-support-policy-for-net-framework-apm).
+The .NET Tracer supports automatic and custom instrumentation on the following .NET Framework versions. It also supports [.NET Core][2].
+
+| .NET Framework Version  | Microsoft End of Life | Support level                       | Package version             | Datadog End of Life |
+| ----------------------- | --------------------- | ----------------------------------- | --------------------------- | ------------------- |
+| 4.8                     |                       | [GA](#support-ga)                   | latest                      |                     |
+| 4.7.2                   |                       | [GA](#support-ga)                   | latest                      |                     |
+| 4.7                     |                       | [GA](#support-ga)                   | latest                      |                     |
+| 4.6.2                   |                       | [GA](#support-ga)                   | latest                      |                     |
+| 4.6.1                   | 04/26/2022            | [GA](#support-ga)                   | latest                      |                     |
+| 4.6                     | 04/26/2022            | [EOL](#support-eol)                 | < 2.0.0 (e.g. [1.31.2][3]) | 04/26/2022          |
+| 4.5.2                   | 04/26/2022            | [EOL](#support-eol)                 | < 2.0.0 (e.g. [1.31.2][3]) | 04/26/2022          |
+| 4.5.1                   | 01/12/2016            | [EOL](#support-eol)                 | < 2.0.0 (e.g. [1.31.2][3]) | 04/26/2022          |
+| 4.5                     | 01/12/2016            | [EOL](#support-eol)                 | < 2.0.0 (e.g. [1.31.2][3]) | 04/26/2022          |
+
+Additional information can be found within [Microsoft's .NET Core Lifecycle Policy][4] and in [Runtime support policy for .NET Framework APM](#runtime-support-policy-for-net-framework-apm).
+
+<div class="alert alert-info">
+  <div class="alert-info"><b>Note:</b> If you use automatic instrumentation only, the .NET Framework Version to take into account is the version installed on the application server.<br/>
```

Comment:
Hmm thanks for the suggestion. But this do not reflect what we want to say here. We could replace it by : "the .Net Framework version you need to consider to decide what tracer to use is the one installed on the application server" ? Not sure it's better though

Reply:
Essentially, what we are trying to say is "The .NET Framework Version that you should be looking at in this table is the version that is installed on the application server". Maybe we can reword this slightly and it would work?

Reply:
How about: "When deciding which tracer version to use for an automatic instrumentation, use the .NET Framework version installed on the application server."